vmdk_plugin/Makefile

```diff
@@ -294,13 +294,14 @@ deploy: deploy-all
 .PHONY: test
 test:
 	$(log_target)
-	$(SCRIPTS)/build.sh testasroot
+	$(SCRIPTS)/build.sh testasroot "$(ESX)" "$(VM1)" "$(VM2)"
 
 .PHONY: testasroot
 testasroot:
```

Comment:
Thanks @kerneltime ! 

Added a new target and here is the output for your reference:

```
nshukla-mbp:docker-volume-vsphere nshukla$ make e2e-tests
/Library/Developer/CommandLineTools/usr/bin/make --directory=vmdk_plugin e2e-tests

=> Running target e2e-tests Mon Jan 9 11:33:28 PST 2017

../misc/scripts/build.sh e2e-dkrVolDriver-test "10.20.105.97" "10.20.105.77" "10.20.105.89"
make: Entering directory `/go/src/github.com/vmware/docker-volume-vsphere/vmdk_plugin'

=> Running target e2e-dkrVolDriver-test Mon Jan 9 19:33:28 UTC 2017

GO15VENDOREXPERIMENT=1 go test github.com/vmware/docker-volume-vsphere/vmdk_plugin/E2E_Tests -cover -v
=== RUN   TestAccessOption
```